backends/ipvs-as-sink/clusteripsvc.go

```diff
@@ -18,158 +18,128 @@ package ipvssink
 
 import (
 	"bytes"
+	"k8s.io/klog"
 	"sigs.k8s.io/kpng/api/localnetv1"
 )
 
-func (s *Backend) handleClusterIPService(svc *localnetv1.Service, op Operation) {
-	key := svc.Namespace + "/" + svc.Name
+func (s *Backend) updateClusterIPService(svc *localnetv1.Service, serviceIP string, port *localnetv1.PortMapping) {
```

Comment:
I see you are renaming the function to make easy understanding but I can't see you updating:
```
backends/ipvs-as-sink/ipvs.go:267:		s.handleClusterIPService(svc, AddService)
backends/ipvs-as-sink/ipvs.go:287:		s.handleClusterIPService(svc, DeleteService)
```

Reply:
if you see `handleUpdatedClusterIPService` , it handles updation of port.